registration/validators.py

```diff
@@ -40,3 +40,27 @@ def validate_student(student):
 def validate_mc_options(field):
     if not all(isinstance(option, str) for option in field.options):
         raise ValidationError("One of the provided options is not a string")
+
+
+def validate_client_interaction(interaction):
+    from enrolments.validators import validate_schema
+
+    INTERACTION_SCHEMA = {
+        "type": "str",
+        "date": "str",
+        "user_id": "int",
+        "user_email": "str",
```

Comment:
i don't think we need the email for anything, so we can remove this! i know the ticket originally said the name too, but i think the frontend will have id/name mappings as a context so it's all good